### ISSUE TITLE
MIT or LGPL license is reported as MIT

### ISSUE THREAD

**Issue by majduddin** [davglass/license-checker#166]
The xmldom module's license file says that you can chose anyone of these: MIT or LGPL.

https://github.com/jindw/xmldom/blob/master/LICENSE

The license-checker tool reports it as LGPL. Shouldn't it be reporting MIT instead?

Thanks.

**Comment by davglass** [comment]
Because the published version is different than the one in the repo:

http://registry.npmjs.org/xmldom/latest

```js
licenses: [{
    type: "LGPL",
    url: "http://www.gnu.org/licenses/lgpl.html",
    MIT: "http://opensource.org/licenses/MIT"
}]
```
